src/status_im/contexts/wallet/wallet_connect/core.cljs

```diff
@@ -158,6 +158,17 @@
                 (not (:watch-only? %)))
           accounts))
 
+(defn latest-connected-account-address
+  [sessions]
+  (let [all-accounts (->> sessions
+                          (sort-by :expiry >)
```

Comment:
do we have any "issuance" date instead of "expiry", since if dapps can add custom expiration dates in the future, this will be broken. Not a big deal tho, just wondering if we have the issuance field available.